cli/azd/pkg/project/internal/project_utils.go

```diff
@@ -34,3 +37,25 @@ func CreateDeployableZip(appName string, path string) (string, error) {
 
 	return zipFile.Name(), nil
 }
+
+const c_gitIgnore string = ".gitignore"
```

Comment:
Consider something like `cGitIgnoreFileName`?  We don't use the underscore like this much.